addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -856,7 +881,7 @@ action = act
         <field name="name">hr.recruitment.source.tree</field>
         <field name="model">hr.recruitment.source</field>
         <field name="arch" type="xml">
-            <tree string="Sources of Applicants" editable="top" class="o_recruitment_list">
+            <tree string="Sources of Applicants" editable="top" class="o_recruitment_list width_class">
```

Comment:
As specified in the task pad, let's not do that.
We will let the JS framework optimize the column width.